app/monitoringapi_internal_test.go

```diff
@@ -136,13 +136,11 @@ func TestStartChecker(t *testing.T) {
 			advanceClock(clock, 10*time.Second, 2)
 
 			// Advance clock for first epoch tick.
-			advanceClock(clock, 384*time.Second, 2)
-
-			// Advance clock for last tick.
-			advanceClock(clock, 10*time.Second, 2)
+			advanceClock(clock, 32*12*time.Second, 2)
 
 			if tt.err != nil {
 				require.Eventually(t, func() bool {
+					advanceClock(clock, 10*time.Second, 2)
```

Comment:
@dB2510 this was flapping. When retrying, you need to advance the clock, otherwise nothing happens and retries do nothing always returning the same result.